### PR TITLE
Resolved issue #859. Fixed attribute error.

### PR DESCRIPTION
**Pull request by Nandika-A**
The error was caused at /events/eventname/applications due to selecting more than one way of ordering. 
It is resolved by giving a default value to getattr(), obtained from the function get_default(). This function slices the string to get the first order selected.

### CODE REVIEW

File: applications/services.py [hunk 1]
```diff
@@ -3,6 +3,10 @@
 from applications.models import Application, Event, Score
 from core.models import User
 
+def get_default(order):
```
Comment (marksweb) [inline applications/services.py:6]: Could you please add a docstring that explans what this is for & what it is doing?

**Review by marksweb — CHANGES_REQUESTED** [review]

File: applications/services.py (hunk 1 above)
Comment (Nandika-A) [reply applications/services.py:6]: Ya sure!

**Review by marksweb — APPROVED** [review]
This looks straightforward enough. Would be ideal to have some testing with it though.

**Comment by Nandika-A** [comment]
Hello @amakarudze! I have resolved the issues with flake8. Could you please check it now?

File: requirements.txt [hunk 2]
```diff
@@ -1,331 +1,254 @@
-#
-# This file is autogenerated by pip-compile with Python 3.10
-# by the following command:
-#
-#    pip-compile --output-file=requirements.txt --resolver=backtracking requirements.in
-#
+absl-py==1.4.0
```
Comment (marksweb) [inline requirements.txt:1]: It looks like these changes having been done using `pip-tools` is that correct?

`pip-compile` should be used to generate, and update the project requirements.

**Review by marksweb — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 2 above)
Comment (Nandika-A) [reply requirements.txt:1]: Yes, I used pip-tools to update the virtual environment according to requirements as per the errors shown by flake8. Should I revert the changes in requirements.txt?

Comment (Nandika-A) [reply requirements.txt]: I have restored requirements.txt. Could you please check it now?

Comment (marksweb) [reply requirements.txt]: Thanks. It's not a problem to update the packages. It's just that it should have the comment about it being a generated file and then the packages that use each defined package underneath them.

**Review by marksweb — APPROVED** [review]

**Comment by amakarudze** [comment]
Thanks @Nandika-A. Now merging this in.

**Pull request merged**